mysql-test/suite/galera/r/galera_gcache_recover_manytrx.result

```diff
@@ -131,9 +130,9 @@ DROP PROCEDURE update_simple;
 DROP PROCEDURE insert_1k;
 DROP PROCEDURE insert_1m;
 connection node_1;
-call mtr.add_suppression("Error in Log_event::read_log_event\(\):.*");
+call mtr.add_suppression("Error in Log_event::read_log_event.*");
```

Comment:
The trailing `.*` is actually redundant. I think that passing a `\` to the regular expression layer would require escaping it as `\\`. The `\(\)` would actually be equivalent to `()`, which in PCRE refers to a group, so originally this would have matched `…read_log_event:` while the actual output is `…read_log_event():` So, either the literal parentheses have to be omitted, or represented correctly as `\\(\\)`.